dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/CollectionUtils.java

```diff
@@ -103,7 +109,8 @@ public static boolean isEmpty(Collection coll) {
             return emptyMap;
         }
         String[] strings = str.split(separator);
-        Map<String, String> map = new HashMap<>(strings.length);
+        int initialCapacity = (int)(strings.length / DEFAULT_LOAD_FACTOR) + 1;
+        Map<String, String> map = new HashMap<>(initialCapacity);
```

Comment:
keep old-style maybe better

Reply:
This change is to reduce the automatic scaling of the HashMap to get the maximum initial capacity based on the specified storage capacity and load factor.

Reply:
you are right~

Reply:
you are right~